config/routes.rb

```diff
@@ -19,6 +19,10 @@
   post '/checkout' => 'orders#create'
   get '/checkout' => 'orders#edit'
 
+  namespace :admin do
+    get '/dashboard' => 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.